apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/inline-support-link.tsx

```diff
@@ -22,16 +33,19 @@ export default function DescriptionSupportLink( { children, title, url }: Props
 			{ children }
 			<br />
 			<ExternalLink
-				onClick={ () => {
+				onClick={ ( e ) => {
+					e.preventDefault();
```

Comment:
Before preventing default, we should check that `setShowHelpCenter` is defined. Because some users have ETK disabled and this will throw for them.

Reply:
Added check for setShowHelpCenter and setShowSupportDoc before using triggering preventDefault and help center functions.